src/services/dashboards/widgets/monthly-cost/MonthlyCostWidget.vue

```diff
@@ -168,34 +155,39 @@ const drawChart = (chartData: XYChartData[]) => {
     });
 
     const seriesSettings = {
-        name: 'usd_cost',
-        valueYField: 'usd_cost',
-        valueXField: 'date',
+        valueYField: 'date',
```

Comment:
It would be better make this constant! like `DATE_FIELD_NAME`?!

Reply:
I think the field name is wrong while using the getRefinedXYChartData function, so I'll fix it! Thanx